src/anim/controller/anim-controller.js

```diff
@@ -420,6 +420,9 @@ class AnimController {
     }
 
     removeNodeAnimations(nodeName) {
+        if (ANIM_CONTROL_STATES.includes(nodeName)) {
```

Comment:
`Array#includes` doesn't exist in IE11. Have you checked the Babel output to see if it's IE11-compatible?